adafruit_platformdetect/chip.py

```diff
@@ -42,7 +76,7 @@ def _linux_id(self):
             vendor_id = self.detector.get_cpuinfo_field("vendor_id")
             if vendor_id in ("GenuineIntel", "AuthenticAMD"):
                 linux_id = GENERIC_X86
-        elif hardware in ("BCM2708", "BCM2708", "BCM2835"):
+        elif hardware in ("BCM2708", "BCM2708", "BCM2709", "BCM2835"):
```

Comment:
looks like originally i duplicated `BCM2708` here instead of adding `BCM2709` and it's still duplicated?

Reply:
oh right - yes ok ill remove the second 2708